src/docs/meta/relay/getting-started.mdx

```diff
@@ -161,13 +161,13 @@ directory in which it can find the configuration and credentials files
 For example, you can start the latest version of `relay` as follows:
 
 ```shell
-❯ docker run -v $(pwd)/configs/:/work/.relay/ getsentry/relay run
+# Generate configuration
+docker run -it -v $(pwd)/configs/:/work/.relay/ getsentry/relay config init
```

Comment:
I find it non-intuitive that we don't provide an instruction to generate configuration when going the Docker route. 

It's much more straightforward to provide both commands together (here or elsewhere), it reduces the getting-started with Docker to a few seconds instead of minutes. 

> Separately, if we remove the ❯, then we should remove it in every example.

Ok, but can it be perhaps made non-selectable? 

> For my education, is -it necessary here?

It's necessary for `config init`, otherwise stdin/tty are not connected properly, and you cannot use the interactive menu.